packages/zoe/src/zoe.js

```diff
@@ -212,14 +223,23 @@ import { makeTables } from './state';
  * queries based on other information, we choose to omit it. For
  * instance, `installationHandle` can be derived from
  * `instanceHandle` and is omitted even though it is useful.
- * @param  {object} seat - an object defined by the smart
+ * @param  {OfferHook} offerHook - a function that will be handed the seat at
+ * the right time, and returns a contract-specific value that will be put in the OfferResultRecord.
```

Comment:
The contract-specific value is now the `ContractAPI` or (better) the `OfferOutcome`.

Reply:
addressed in 43aa15b1